apoc/_table_row_classifier.py

```diff
@@ -155,16 +168,39 @@ def _prepare_feature_table(
             The table from which to make the prediction. Each row of the table
             will be classified. The table can either be a pandas DataFrame or a
             Dict with string keys (column names) and numpy array columns.
+        ground_truth : np.array
+            The array containing the ground truth class for each row in feature_table
```

Comment:
I think it would be helpful to add a note that `ground_truth == 0 ` is dropped.

Similarly, a note for the `feature_table` param that `nan` is dropped (possibly linking to the pandas docs for `dropna` so people know how nan is defined in this context.

Finally, can you please add the new `updated_ground_truth` array to the `Returns`